.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "env": {
+    "browser": true
+  },
+  "extends": ["eslint-config-airbnb-base", "prettier"],
+  "plugins": ["prettier"],
+  "rules": {
+    "prettier/prettier": "error",
+    "no-unused-vars": "warn",
+    "no-console": "off",
+    "func-names": "off",
+    "no-process-exit": "off",
+    "object-shorthand": "off",
+    "class-methods-use-this": "off",
```

Comment:
didn't find any classes in code... perhaps we don't need rules `class-methods-use-this: off`.  I suggest check and remove all unused eslint rules